src/OrchardCore.Modules/OrchardCore.Seo/GraphQL/SeoMetaPartIndexAliasProvider.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.Collections.Generic;
+using OrchardCore.ContentManagement.GraphQL.Queries;
+using OrchardCore.Seo.Indexes;
+
+namespace OrchardCore.Seo.GraphQL;
+
+public class SeoMetaPartIndexAliasProvider : IIndexAliasProvider
```

Comment:
Is it used in your custom code to retrieve the Indexes available to Query?

Reply:
Should I remove this? I just saw it being used in AutoPart for indexing thats why I created one for SeoMeta too

Reply:
You don't need to index these. There is no reason to query a content item by these properties IMO.